pkg/migrations/internal/seq_num.go

```diff
@@ -4,7 +4,8 @@ var (
 	// CurrentDBVersionSeqNum is the current DB version number.
 	// This must be incremented every time we write a migration.
 	// It is a shared constant between central and the migrator binary.
-	CurrentDBVersionSeqNum = 112
-	// PostgresDBVersionPlus is the current DB version number with Postgres DB data migration.
-	PostgresDBVersionPlus = 56
+	CurrentDBVersionSeqNum = 168
```

Comment:
Thank you for this semantic change. I think it makes things easier not to have to compute what the `PostgresDBVersionPlus` should be.